learning/tour-of-beam/frontend/integration_test/tour_page_test.dart

```diff
@@ -100,6 +102,11 @@ Future<void> _checkNode(UnitModel node, WidgetTester wt) async {
   );
 
   await _checkUnitContentLoadsProperly(node, wt);
+
+  final hasSnippet = _getTourNotifier(wt).isUnitContainsSnippet;
+  if (!_isSdkChangingChecked && hasSnippet) {
+    await _checkSdkChangesProperly(wt, node);
+  }
```

Comment:
People would not expect `_checkNode` to go into such extra work depending on the global state. And if it breaks, it will be confusing, because it has nothing to do with 'check node'.

Move this to a separate function on the same level as other globals.